client/src/app.jsx

```diff
@@ -1,39 +1,57 @@
 import React from 'react';
-import Container from 'react-bootstrap/Container'
-import Jumbotron from 'react-bootstrap/Jumbotron'
-import Button from 'react-bootstrap/Button';
-import Col from 'react-bootstrap/Col';
-import Row from 'react-bootstrap/Row';
+import { Container, Row } from 'react-bootstrap';
+import TitleImage from './components/titleImage';
+import MainImage from './components/mainImage';
+// import listProducts from './lib/routes.js';
 
-// const apiTest = () => {
-//   fetch('http://52.26.193.201:3000/products/list')
-//     .then(data => {
-//       console.log(data);
-//     })
-//     .catch(err => {
-//       console.log(err);
-//         })
-// }
-const App = () => (
-  <Container>
-    <Jumbotron>
-      <h1>The Proto Company</h1>
-      <h2>It is my app!</h2>
-      <p>
-        This is a simple hero unit, a simple jumbotron-style component for calling
-        extra attention to featured content or information.
-      </p>
-      <p>
-        <Button variant="primary">Learn More</Button>
-      </p>
-    </Jumbotron>
-    <Container>
-      <Row>
-        <Col>
-        </Col>
-      </Row>
-    </Container>
-  </Container>
-);
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+    // this.getProducts = this.getProducts.bind(this);
+  }
+  // getProducts((error, data) => {
+  //   if (error) {
+  //     throw(error)
+  //   } else {
+  //     console.log(data.results);
+  //   }
+  // })
+
+  render() {
+    return (
+      <Container>
+        <br />
```

Comment:
I was wondering how to space out components; padding/margin/br. Do you often use <br>s? Purely inquisitive!

Reply:
I do use this often, usually with divs but I've been experimenting with Container a ton